### ISSUE TITLE
Missing config file results in exception not being sent to Sentry

### ISSUE THREAD

**Issue by nmfzone** [getsentry/sentry-laravel#163]
- Laravel Version: 5.7.11
- PHP Version: 7.2.11
- Database Driver & Version: Mysql 14.4

### Description:
The exception not being sent to the sentry. The problem actually happen on my production app with Laravel 5.6, using sentry-laravel version 0.9.*. Then, I decided to upgrade senty-laravel to 0.10.1. But, the issue still persist.

Finally, I try to use on fresh Laravel installation. But, the issue still.

### Steps To Reproduce:
1. Create new Laravel project
`composer create-project laravel/laravel MyApp --prefer-dist`
2. Add sentry to the app
`composer require sentry/sentry-laravel`
3. Set the `SENTRY_LARAVEL_DSN` to `.env`
4. Add sentry to log channels
```
'channels' => [
    // ...
    'sentry' => [
        'driver' => 'sentry',
    ],
],
```
5. Add sentry channel to stack channel
```
'stack' => [
    'driver' => 'stack',
    // Add the Sentry log channel to the stack
    'channels' => ['single', 'sentry'],
],
```

**Comment by nmfzone** [comment]
Oke, finally I found the problem. The problem is on the `SentryLaravelServiceProvider`.
There is no `mergeConfigFrom()` call in that service provider. That's the key. Because when the user not publish the sentry config file, set `SENTRY_LARAVEL_DSN` on the `.env` is useless.

The simplest solution to solve this issue, instead set `SENTRY_LARAVEL_DSN` on the `.env`, you need to set `SENTRY_DSN` on the `.env`.

**Comment by stayallive** [comment]
Okay good one, need to check when the merge config was introduced so we can implement it.

However, I do want to note that in the installation docs creating the config is mentioned and not as optional :)

**Comment by nmfzone** [comment]
@stayallive Oh nice, my bad. I used Sentry long time ago, and I don't remember at that time, creating (publish) the config is mentioned in docs or not. And, the important thing, before this package introduce `SENTRY_LARAVEL_DSN`, I used only `SENTRY_DSN` without creating config file, all works fine.

Fyi, as far as I know, publish config is always optional on Laravel package. So, maybe `mergeConfigFrom()` should be considered. Thanks
